### ISSUE TITLE
17.11 CE: Cannot create ingress network

### ISSUE THREAD

**Issue by cveld** [moby/moby#35662]
Docker Swarm is not working. I pinpointed it to the following command which fails:

`docker network create -d overlay ingress --ingress`

In the Windows event log I see:

`Failed creating ingress network: HNS failed with error : The parameter is incorrect. `

**Steps to reproduce the issue:**
1. Run `docker network create -d overlay ingress --ingress`
2. Review the Windows event log

**Describe the results you received:**
`docker network ls` reports a newly created overlay ingress network `ingress`.
But in the Hyper-V settings this adapter is not found.
And in the Windows log you will see: `Failed creating ingress network: HNS failed with error : The parameter is incorrect. `
Newly services attached to this swarm are not reachable.

**Describe the results you expected:**
No error in the event log. And services should be reachable.

**Output of `docker version`:**

```
Docker version 17.11.0-ce, build 1caf76c
```

**Output of `docker info`:**

```
Containers: 7
 Running: 0
 Paused: 0
 Stopped: 7
Images: 2
Server Version: 17.11.0-ce
Storage Driver: windowsfilter (windows) lcow (linux)
 LCOW:
Logging Driver: json-file
Plugins:
 Volume: local
 Network: ics l2bridge l2tunnel nat null overlay transparent
 Log: awslogs etwlogs fluentd json-file logentries splunk syslog
Swarm: active
 NodeID: nxzn1pm16jbkltxedyawbb1zt
 Is Manager: true
 ClusterID: veq3hfm9v2eicpksdz438fvik
 Managers: 1
 Nodes: 1
 Orchestration:
  Task History Retention Limit: 5
 Raft:
  Snapshot Interval: 10000
  Number of Old Snapshots to Retain: 0
  Heartbeat Tick: 1
  Election Tick: 3
 Dispatcher:
  Heartbeat Period: 5 seconds
 CA Configuration:
  Expiry Duration: 3 months
  Force Rotate: 0
 Autolock Managers: false
 Root Rotation In Progress: false
 Node Address: 192.168.1.16
 Manager Addresses:
  192.168.1.16:2377
Default Isolation: hyperv
Kernel Version: 10.0 16299 (16299.15.amd64fre.rs3_release.170928-1534)
Operating System: Windows 10 Pro
OSType: windows
Architecture: x86_64
CPUs: 4
Total Memory: 31.7GiB
Name: masked
ID: KNFJ:SN2F:VZGO:3FUJ:NWMR:RMBY:FQV7:NLIS:YSVP:ZPFI:L6TI:4YQS
Docker Root Dir: C:\ProgramData\Docker\lcow
Debug Mode (client): false
Debug Mode (server): true
 File Descriptors: -1
 Goroutines: 134
 System Time: 2017-12-01T10:50:31.3771932+01:00
 EventsListeners: 1
Registry: https://index.docker.io/v1/
Experimental: true
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**
Windows 10 Version 1709
Running Docker 17.11 CE in native Linux Container mode.
When running in "classic" mode everything works fine.

**Comment by msabansal** [comment]
Asking the obvious question, did you run docker swarm init? Please collect following logs and share: 

netsh trace start globallevel=5 provider=Microsoft-Windows-Host-Network-Service level=5 report=di

**Comment by jonashackt** [comment]
I had the same issue while deploying a Docker Swarm service based on a native Docker Windows Container on Windows Server 2016 Core/Datacenter 1709 (Fall Creators Update): 

`HNS failed with error : The parameter is incorrect.`

After lots of experiments I finally installed the latest Updates via [PSWindowsUpdate](https://gallery.technet.microsoft.com/scriptcenter/2d191bcd-3308-4edd-9de2-88dff796b0bc), especially the Novembers cumulative Update package, including the [KB4048955](https://support.microsoft.com/en-us/help/4048955):

<img width="1010" alt="working-build-nr-and-update-kb4048955-for-ingress-networking-docker-windows" src="https://user-images.githubusercontent.com/6949966/33716338-16daaa80-db56-11e7-851d-e9aeaff8e2e1.png">

And now the error is gone and I´am finally able to deploy the native Docker Windows Image as a Docker Swarm service:

<img width="1023" alt="first-successful-docker-windows-ingress-swarm-service" src="https://user-images.githubusercontent.com/6949966/33716329-0b3d2dce-db56-11e7-9f51-9dc7d39670d9.png">

**TLDR: Try to install the newest updates!**

**Comment by thaJeztah** [comment]
Closing as this went stale
